image_width: 752
image_height: 480
camera_name: camera_9
camera_matrix:
  rows: 3
  cols: 3
  data: [630.3761156230871, 0.0, 397.2942696299946, 0.0, 617.2866321376528, 208.324427257397, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.2904290929437095, 0.10986446592729726, 0.01817109105463184, -0.0035875167202699936, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.9987929471731123, -0.0035841437163768508, 0.04898778001776343, 0.0041448485410405704, 0.9999270073019092, -0.01134902193228248, -0.04894352774188086, 0.011538369991812148, 0.9987348983139178]
projection_matrix:
  rows: 3
  cols: 4
  data: [608.8018361191206, 0, 363.6506500244141, -63.94820257264251, 0, 608.8018361191206, 223.6016578674316, 0, 0, 0, 1, 0]